addons/portal/static/src/js/portal_chatter.js

```diff
@@ -116,6 +110,7 @@ var PortalChatter = publicWidget.Widget.extend({
         }).then(function (result) {
             self.set('messages', self.preprocessMessages(result['messages']));
             self.set('message_count', result['message_count']);
+            return result;
```

Comment:
Now that I look at it again, it doesn't seem like this result is used anywhere, why are we returning it?

Reply:
as we have changed messagefetch method to update rating card view, we need to return rating_stats to _updateRatingCardValues, which will set rating_stats, and in turn rating card will be updated.

Reply:
here, we used result value for update rating card details https://github.com/odoo/odoo/pull/47746/files#diff-c1b81e8fefbde103bb78c40b5b5db37cR130

Reply:
Right, seems good then! :+1: